frontend/public/reducers/features.ts

```diff
@@ -65,9 +59,6 @@ export const featureReducer = (state: FeatureState, action: FeatureAction): Feat
 
   switch (action.type) {
     case ActionType.SetFlag:
-      if (!isKnownFlag(action.payload.flag)) {
-        throw new Error(`unknown flag ${action.payload.flag}`);
```

Comment:
Originally, this code logged a warning when trying to set a flag whose name is not in the static `FLAGS` enum.

This code was then modified (#3987) to support additional flag names, driven by `ext.properties.flag` value.

I'm OK with removing this check, so just a heads-up.